### DISCUSSION TITLE
Data Split for BBBC006 Dataset

### DISCUSSION THREAD

**Discussion by caijd2000** [PieceZhang/MPT-CataBlur#2] [Q&A]
I have been reading your paper with great interest, particularly the methodology you've employed. I am keen on replicating the code for further study and analysis. However, I encountered a point of confusion regarding the data set division for the BBBC006 dataset.
In the paper, you mention that "images with z-stack = [2, 6, 10] are set as blurry input for training." This is quite clear, but I was unable to find explicit information on which specific data was allocated to the validation and test sets.
Thank you for your time and assistance.

**Comment by PieceZhang** [comment]
> I have been reading your paper with great interest, particularly the methodology you've employed. I am keen on replicating the code for further study and analysis. However, I encountered a point of confusion regarding the data set division for the BBBC006 dataset. In the paper, you mention that "images with z-stack = [2, 6, 10] are set as blurry input for training." This is quite clear, but I was unable to find explicit information on which specific data was allocated to the validation and test sets. Thank you for your time and assistance.

Hi,
Please refer to the data description given by the author of the BBBC006 dataset, where the name of sharp (GT) and blurry (training) images are provided. Thank you.

**Comment by caijd2000** [comment]
Thank you for your prompt response. 
I reviewed the dataset description provided at (https://bbbc.broadinstitute.org/BBBC006), and from my understanding there was no specific delineation of training and validation sets. I understand that images with z-stack = [2, 6, 10] are designated as blurry, while those at z-stack = 16 are considered Ground Truth (GT). However, it remains unclear to me whether the metrics in Table 1 were derived solely from the z-stack = [2, 6, 10] images or if other z-stacks were also utilized.
With the intention of referencing your study in my own research and ensuring a rigorous experimental comparison, I kindly request further clarification. Thanks.
